frontend/src/components/SortServices.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { useEffect } from 'react'
+import { Badge, IconButton, Menu, MenuItem } from '@material-ui/core'
+import { Icon } from './Icon'
+import { useDispatch } from 'react-redux'
+import { Dispatch } from '../store'
+import { makeStyles } from '@material-ui/core/styles'
+import { fontSize } from '@remote.it/components/lib/styles/variables'
+
+export interface ISortService {
+  value: number
+  name: string
+  sortService: (a: IService, b: IService) => number
+}
+const options: ISortService[] = [
```

Comment:
Let's export this so it can be referenced by the devicesPage.tsx file.

Also, instead of an array it could be a lookup. For example:
```
{
   ATOZ: { sort: ..., name: ..., icon: 'up' },
   ZTOA: ...
   NEWEST: ...
   OLDEST:...
}
```
That way when you save the selected sort state  you just write the ENUM to the model:
`devices.set({ serviceSort }: { serviceSort: 'ATOZ' | 'ZTOA' | 'NEWEST' | 'OLDEST' })`